swarm/pss/client/client_test.go

```diff
@@ -24,6 +25,7 @@ func init() {
 }
 
 func TestRunProtocol(t *testing.T) {
+	t.Skip("pssclient is broken, needs whisper integration")
```

Comment:
yes but it makes sense to evaluate the encryption stuff first. If we keep it, then I'll go ahead and fix the client.